test/promisified/admin/fetch_topic_metadata.spec.js

```
@@ -0,0 +1,246 @@
+jest.setTimeout(30000);
```

Add another test for fetching a non existent topic, both fetching on its own, and fetching in a list of one valid topic, and one non-existent one. Both should throw the same error.